backend/Recodo.API/Controllers/UserController.cs

```diff
@@ -58,5 +58,17 @@ public async Task<IActionResult> GetUserFromToken()
 
             return Ok(user);
         }
+
+
+        [HttpGet]
+        [AllowAnonymous]
+        [Route("AddToTeam/{authorEmail}")]
```

Comment:
again, non readable /**addtoteam**/{authorEmail}
and I don't like this authorEmail. why is it called like this?

```suggestion
        [Route("add-to-team/{authorEmail}")]
```